azure/v12.1.0/README.md

```diff
@@ -1,8 +1,10 @@
 ## :zap:  Giant Swarm Release 12.1.0 for Azure :zap:
 
+***We recommended triggering this upgrade by our solution engineers to mitigate any unexpected results.***
+
 **If you are upgrading from 12.0.2, upgrading to this release will not roll your nodes. It will only update the apps.**
 
-This release upgrades all Helm releases managed by Giant Swarm to use Helm v3.3.0.
+This release upgrades all Helm releases managed by Giant Swarm to use [Helm v3.3.0](https://github.com/helm/helm/releases/tag/v3.3.0).
 
 This lets us benefit from the improved security model and keep up to date with the community. We also remove the Tiller deployment from the giantswarm namespace, removing its gRPC endpoint, which reduces operational complexity.
 
```

Comment:
```suggestion
**Note before upgrade:**

Please contact your Solution Engineer before upgrading. The upgrade is automated. However, it includes a data migration from Helm 2 release configmaps to Helm 3 release secrets, which we recommend to monitor to ensure safety.
```

@rossf7 in previous releases, we've been adding notes after the paragraph. SEs might be used to finding them in this section. See this [example](https://github.com/giantswarm/releases/tree/master/aws/v12.1.4). Also, I suggested some wording changes.

Reply:
I think I put this in the wrong section :/

Reply:
@cokiengchiara Thanks I like that example much better. @tomahawk28 can you update to use that format?

@pipo02mix Could you review once we've made these changes?

Reply:
Made the changes here. https://github.com/giantswarm/releases/pull/432/files#r481931625